frontend/coprs_frontend/coprs/views/misc.py

```diff
@@ -89,6 +89,64 @@ def oid_login():
     return FedoraAccounts.login()
 
 
+if OpenIDConnect.enabled(app.config):
```

Comment:
I'm not sure about this "global" module-level code flow.  We should do this minimally, and we typically do this in `frontend/coprs_frontend/coprs/__init__.py`  (with lots of ugly hacks).  Would you mind moving it there?  Perhaps wrapping the logic with something like  `enable_oidc_authentication(app)` (implemented in auth.py?) and just calling it in `__init__.py`?